releasenotes/notes/fix-patching-error-asynctest-9b430f410cd9dfeb.yaml

```diff
@@ -1,5 +1,5 @@
 ---
 fixes:
```

Comment:
maybe worth having an "other" note too for saying that we are vendoring the dependency (in case someone would have it pinned or something)